client/elasticsearch/v710/client.go

```diff
@@ -153,6 +154,7 @@ func (cli *ESClient) DeleteIndices(ctx context.Context, indices []string) error
 //
 // See full documentation at https://www.elastic.co/guide/en/elasticsearch/reference/master/search-count.html.
 func (cli *ESClient) Count(ctx context.Context, count client.Count) ([]byte, error) {
+	log.Info(ctx, "call to count api with count query", log.Data{"countQuery": count})
```

Comment:
So the log library shouldn't be used in this library;

This info log can be added to the application before calling Count method if you are trying to check the count query

Reply:
Fixed in update1.